esti/multipart_test.go

```diff
@@ -44,6 +52,11 @@ func TestMultipartUpload(t *testing.T) {
 	}
 
 	completedParts := uploadMultipartParts(t, ctx, logger, resp, parts, 0)
+
+	// On S3 the object should have Last-Modified time at around time of MPU creation.
+	// Ensure lakeFS fails the test if it fakes it by using the current time.
+	time.Sleep(2 * timeResolution)
```

Comment:
Never too happy about hardcoded sleeps in tests but seems inevitable.
Perhaps scope it to blockstoreType == s3?

Reply:
I completely understand.  Don't like it but did it, thanks!